CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Upcoming version
 
+- [#509]
+  - **Description:** Introduces `appearanceOverrides` prop for the `KImg` component
+  - **Products impact:** new API 
+  - **Addresses:** https://github.com/learningequality/kolibri-design-system/issues/371
+  - **Components:** KImg
+  - **Breaking:** No
+  - **Impacts a11y:** No
+  - **Guidance:** -
```

Comment:
Thank you for being diligent and providing guidance part @EshaanAgg. I removed it in my before merge commit because we rather want to discourage using this prop and it doesn't affect a11y directly as much as was indicated in your comment. It's main purpose is to allow flexible updates of styles when the implemented API is not sufficient, but overall we don't want to rely on it much.